demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -26,6 +30,22 @@ class IntegrationScript(ContentItem):
     code: Optional[str] = Field(None, exclude=True)
     unified_data: dict = Field(None, exclude=True)
 
+    @lazy_property
+    def python_version(self) -> Optional[str]:
+        """
+        Get the python version from the script/integration docker-image in case it's a python image
+        """
+        if "python" not in self.type:
+            logger.debug(
+                f"The {self.content_type} = {self.object_id=} that uses {self.docker_image=} is not a python image"
+            )
+            return None
```

Comment:
if not already handled in `get_python_version`, it should

Reply:
you are absolutely right, this is already handled in `get_python_version`, hence removing it.